### PR TITLE
Clean up prototype pages

### PR DESCRIPTION
**Pull request by anselmbradford**
Clean up prototype pages

## Changes

- Add missing atomic elements to doc detail and learn pages that are already built.
- Minor updating of global_dict format of image and text 25/75 and well.

## Testing

- `/document-detail/` and `/learn-page/`

## Review

- @sebworks 
- @KimberlyMunoz

### CODE REVIEW

File: cfgov/jinja2/v1/browse-basic/index.html [hunk 1]
```diff
@@ -78,10 +78,10 @@
              %}
             {% set value = global_dict.expandable %}
             {% include 'molecules/expandable.html' with context %}
+            {% set value = None if prototype %}
```
Comment (sebworks) [inline cfgov/jinja2/v1/browse-basic/index.html]: Does this need and `else` condition?

**Comment by sebworks** [comment]
:+1:

**Comment by KimberlyMunoz** [comment]
Fixes #1225

**Comment by KimberlyMunoz** [comment]
Also, I notice expandables aren't working after a gulp build. That's fine if it was like that before and out of scope for this PR, but we should make a note if it's an issue.

**Comment by anselmbradford** [comment]
> Also, I notice expandables aren't working after a gulp build. That's fine if it was like that before and out of scope for this PR, but we should make a note if it's an issue.

Yeah, that's broken after https://github.com/cfpb/cfgov-refresh/pull/1253 I believe. @kave would know whether that matters or not for the prototype pages.

File: cfgov/jinja2/v1/browse-basic/index.html (hunk 1 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/browse-basic/index.html]: I'm actually unclear when the prototype pages would be viewed without the `prototype` value being present. The purpose of this is when setting the mock data in the `value` for the prototype pages, if you don't null it out the next atomic element gets the date from the previous atomic element. So like the text intro gets expandable data, etc.

**Pull request merged**